test/app/static_pages_test.rb

```diff
@@ -32,6 +32,12 @@ def test_route_help
     assert_match(/Help/, last_response.body)
   end
 
+  def test_route_help_with_trailing_slash
```

Comment:
I wanted to have two tests for trailing slashes - one with logged-in route and one with public route. Also I removed the `help` redirection from legacy routes that is tested as well.